src/HoldRequestResultConsumerListener.php

```diff
@@ -263,30 +193,31 @@ protected function skipMissingPatron(HoldRequestResult $holdRequestResult)
 
     /**
      * @param Patron $patron
-     * @param array $bibs
-     * @param Item $item
      * @param HoldRequest $holdRequest
      * @param HoldRequestResult $holdRequestResult
      */
     protected function sendEmail(
         Patron $patron,
```

Comment:
it might make sense to extract the notifypatron call from this function and change the name to something like "prepareHoldData" and then have the call to notifyPatron happen as a separate call on line 259. it's not actually sending an email, and it feels like separate enough concerns to warrant splitting them up. 
not a huge issue, obviously, and ultimately up to you!